run_analysis.py

```diff
@@ -41,50 +35,87 @@ def makedirs(inpath):  # {{{
     return inpath  # }}}
 
 
+def check_generate(config, analysisName, mpasCore, analysisCategory=None):
+    # {{{
+    """
+    determine if a particular analysis of a particular core and (optionally)
+    category should be generated.
```

Comment:
Yep, good point.  I was excited about getting to the coding and forgot that I guess.